web/impact/impact/tests/test_api_routes.py

```diff
@@ -111,6 +111,40 @@ def test_api_object_get(self):
             response_dict = json.loads(response.content)
             self.assertIn("is_visible", response_dict.keys())
 
+    def test_api_many_to_many_object_is_displayed(self):
+        StartupFactory(is_visible=1, organization__url_slug="test1")
+        StartupFactory(is_visible=1, organization__url_slug="test2")
+        StartupFactory(is_visible=1, organization__url_slug="test3")
+        Permission.objects.get_or_create(
+            content_type=ContentType.objects.get(
+                app_label='mc',
+                model='startup'),
+            codename='view_startup_additional_industries'
+        )
+        view_perm, _ = Permission.objects.get_or_create(
+            content_type=ContentType.objects.get(
+                app_label='mc',
+                model='startup'),
+            codename='view_startup_additional_industries'
+        )
+        url_name = "object-list"
+        view_kwargs = {'app': 'impact',
+                       "model": "Startup_additional_industries"}
+        self.response_401(self.get(url_name, **view_kwargs))
+
+        basic_user = self.make_user('basic_user@test.com')
+        with self.login(basic_user):
+            self.response_403(self.get(url_name, **view_kwargs))
```

Comment:
The 401/403 asserts seem unrelated to the test you're attempting.

Reply:
This was an adaptation of another test and was pretty complex to figure out.  I'm going to leave this as is unless you feel very strongly about it.